### PR TITLE
fix comment that I missed in the forst PR

### PR DESCRIPTION
**Pull request by szuecs**
Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

**Comment by szuecs** [comment]
:+1:

File: readme.md [hunk 1]
```diff
@@ -234,5 +234,5 @@ Kubernetes ingress controller and runs several custom skipper
 instances that use skipper as library.
 
 Sergio Ballesteros from [spotahome](https://www.spotahome.com/)
-> We also runned tests with serveral ingress controllers options(istio not existed at that time) and skipper gave us the more reliable results. Currently we are running skipper since almost 2 years with like 20K Ingress rules.
-> Envoy is also a good candidate but the fact that skipper is written in go let us understand the code, add features and fix bugs since all of our infra stack is golang.
+> We also runned tests with serveral ingress controllers and skipper gave us the more reliable results. Currently we are running skipper since almost 2 years with like 20K Ingress rules.
```
Comment (mikkeloscar) [inline readme.md]: `runned` -> `ran`

**Comment by szuecs** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**